### PR TITLE
Bump the verbosity of the hybrid-overlay service in debug mode

### PR DESCRIPTION
**Pull request by jrvaldes**
This PR aims to adjust the hybrid-overlay log verbosity according
to the controller-runtime logging level.

### CODE REVIEW

File: pkg/windows/windows.go [hunk 1]
```diff
@@ -496,6 +496,12 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	hybridOverlayServiceArgs := "--node " + nodeName + customVxlanPortArg + " --k8s-kubeconfig c:\\k\\kubeconfig " +
 		"--windows-service " + "--logfile " + hybridOverlayLogDir + "hybrid-overlay.log"
 
+	// check log level and increase hybrid-overlay verbosity if needed
+	if ctrl.Log.V(1).Enabled() {
```
Comment (sebsoto) [inline pkg/windows/windows.go:500]: Please use windows.log for this check

File: pkg/windows/windows.go [hunk 2]
```diff
@@ -496,6 +496,12 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
...
+		// append loglevel param using 5 for debug (default: 4)
+		hybridOverlayServiceArgs = hybridOverlayServiceArgs + " --loglevel 5"
```
Comment (sebsoto) [inline pkg/windows/windows.go:502]: if the default is 4 wouldnt raising it to 5 make it less verbose?

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: pkg/windows/windows.go (hunk 2 above)
Comment (jrvaldes) [reply pkg/windows/windows.go:502]: Will add a reference link to the [code](https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L736) for clarification.

[the remark this answers is not in the document]
Comment (jrvaldes) [reply pkg/windows/windows.go:502]: See [code](https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L736) for details.

File: pkg/windows/windows.go (hunk 1 above)
Comment (jrvaldes) [reply pkg/windows/windows.go]: Addressed in [41fa41a ](https://github.com/openshift/windows-machine-config-operator/commit/41fa41ae3430f6f999fe1c15ddd576d204283dee)

**Comment by jrvaldes** [comment]
/hold cancel 

https://github.com/openshift/windows-machine-config-operator/pull/1102 merged and CI is back to green.

**Pull request merged**